.codeclimate.yml

```diff
@@ -11,4 +11,3 @@ exclude_paths:
 - "Dockerfile"
 - "bin/fixme"
 - "tests/**"
-- ".codeclimate.yml"
```

Comment:
Because https://github.com/codeclimate/codeclimate-fixme/commit/ee8d1cf27b2c9612cd8bc9aa17175dd7eb438167

Essentially we don't need it now. But happy to keep if that seems preferable?